R/build-home-authors.R

```diff
@@ -59,11 +65,18 @@ data_author_info <- function(pkg = ".") {
 
 data_home_sidebar_authors <- function(pkg = ".") {
   pkg <- as_pkgdown(pkg)
-  data <- data_authors(pkg)
+  data <- data_authors(
+    pkg,
+    roles = pkg$meta$authors$sidebar$roles %||% default_roles()
+  )
 
-  authors <- data$main %>% purrr::map_chr(author_desc)
-  if (data$needs_page) {
+  authors <- data$main %>% purrr::map_chr(author_desc, comment = FALSE)
+  if (data$more_authors) {
     authors <- c(authors, "<a href='authors.html'>All authors...</li>")
```

Comment:
or maybe go back to `needs_page` and some general phrasing?

Reply:
Yeah, I think `needs_page` is a bit clearer